README.md

```diff
@@ -120,6 +120,26 @@ class Example {
         person=spark.people().path("/<<<**Insert PersonId**>>>").put(person);
 
 
+        // Create a new webhook
+        Webhook webhook=new Webhook();
+        webhook.setName("My Webhook");
+        webhook.setResource("messages");
+        webhook.setEvent("created");
+        webhook.setFilter("mentionedPeople=me");
+        webhook.setTargetUrl("http://www.example.com/webhook");
```

Comment:
Not a String its URI, please use the following instead webhook.setTargetUrl(URI.create("http://www.example.com/webhook"));